lib/alephant/broker/component.rb

```diff
@@ -35,7 +35,9 @@ def headers
       end
 
       def status
-        data[:meta].key?("status") ? data[:meta]["status"] : 200
+        meta = data.fetch(:meta, {})
+
+        meta[:status] || meta['status'] || 200
```

Comment:
Do we have any way to standardise the keys in the input to be only symbols or strings?